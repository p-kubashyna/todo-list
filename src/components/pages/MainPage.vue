<script setup>
import BackgroundImage from '../../components/atoms/BackgroundImage.vue'
import TodoMainComponent from '../../components/templates/TodoMainComponent.vue'
</script>

<template>
  <BackgroundImage/>
  <div class="content">
  <TodoMainComponent/>
  </div>
  <div><p> dfgdfffffffffffffffffffffffffffffffffffg</p></div>
</template>

<style>

.content {
  position: relative;
  padding: 200px;
}

#app {
  font-family: Avenir, Helvetica, Arial, sans-serif;
  -webkit-font-smoothing: antialiased;
  -moz-osx-font-smoothing: grayscale;
  text-align: center;
  color: #2c3e50
}
</style>
