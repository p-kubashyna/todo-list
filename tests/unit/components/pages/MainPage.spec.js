import { describe, expect, it } from "@jest/globals";
import { shallowMount } from "@vue/test-utils";
import MainPage from "@/components/pages/MainPage.vue";

describe("Given the MainPage component", () => {
  it("should render", () => {
    const wrapper = shallowMount(MainPage, {});

    expect(wrapper.element).toMatchSnapshot();
  });
});
