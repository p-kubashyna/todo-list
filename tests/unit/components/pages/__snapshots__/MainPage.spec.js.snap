// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Given the MainPage component should render 1`] = `<todo-main-component-stub />`;
